crates/holochain/tests/websocket/mod.rs

```diff
@@ -312,6 +312,13 @@ async fn remote_signals() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[tokio::test(flavor = "multi_thread")]
+#[cfg(feature = "slow_tests")]
+#[cfg_attr(target_os = "macos", ignore = "flaky")]
+async fn agent_centric_remote_signals() -> anyhow::Result<()> {
+    todo!("rewrite remote_signals with agent centricity");
```

Comment:
* look at the `remote_signals_batch` test to see how it uses rendezvous instead of the `conductors.exchange_peer_info()` call

Reply:
i've taken note of this